src/Confluent.Kafka/Admin/ElectLeadersReport.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     Represents the result of an elect leaders request (including error status).
+    ///  </summary>
+    public class ElectLeadersReport
+    {
+        /// <summary>
+        ///    Operational Error status.
+        ///  </summary>
+        public Error Error { get; set; }
+
+        /// <summary>
+        ///   Individual partition results.
```

Comment:
```suggestion
        ///    Individual partition results.
        ///    At least one of these will be in error.
```

Reply:
It does not necessary means one of the partitions is in Error we are using the top error code to decide if it is in error and it could be set to no error.

Reply:
if it's no error a Result is returned instead